server/services/curiousService.test.ts

```diff
@@ -30,13 +43,13 @@ describe('curiousService', () => {
       const username = 'a-dps-user'
 
       const systemToken = 'a-system-token'
-      hmppsAuthClient.getSystemClientToken.mockImplementation(() => Promise.resolve(systemToken))
+      hmppsAuthClient.getSystemClientToken.mockResolvedValue(systemToken)
```

Comment:
Not really related to the Curious API and getting education data; but `mockResolved` and `mockRejected` is a better/cleaner approach than `mockImplementation`